src/Symfony/Component/Serializer/NameConverter/ChainNameConverter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\NameConverter;
+
+/**
+ * ChainNameConverter allows to process a property name with multiple converters.
+ *
+ * @author Jérôme Gamez <jerome@gamez.name>
+ */
+class ChainNameConverter implements NameConverterInterface
+{
+    /**
+     * @var NameConverterInterface[]
+     */
+    protected $converters = array();
+
+    /**
+     * @param array $converters A list of name converters that will be used to convert a given property name.
+     */
+    public function __construct(array $converters = array())
+    {
+        foreach ($converters as $converter) {
+            $this->addConverter($converter);
+        }
```

Comment:
Let's change the param type in the docblock to `NameConverterInterface[]`. Then it's the user's responsibility to follow the contract that we make here.